qiskit/visualization/circuit/_utils.py

```diff
@@ -197,22 +196,19 @@ def get_bit_register(circuit, bit):
     return bit_loc.registers[0][0] if bit_loc.registers else None
 
 
-@deprecate_arg("reverse_bits", since="0.22.0", package_name="qiskit-terra")
-def get_bit_reg_index(circuit, bit, reverse_bits=None):
+def get_bit_reg_index(circuit, bit):
```

Comment:
```suggestion
@deprecate_arg(
    "reverse_bits",
    since="0.22.0",
    package_name="qiskit-terra",
    removal_timeline="in the Qiskit 1.0 release",
)
def get_bit_reg_index(circuit, bit, reverse_bits=None):
```